Change to build with the updated gnome-desktop version.  Can be dropped
when gnome-control-center update goes in.

--- a/panels/background/cc-background-item.c	2020-01-21 10:24:22.046786395 -0800
+++ b/panels/background/cc-background-item.c	2020-01-21 10:22:32.818357790 -0800
@@ -172,7 +172,7 @@
                 gdk_color_parse (item->priv->secondary_color, &scolor);
         }
 
-        gnome_bg_set_color (item->priv->bg, item->priv->shading, &pcolor, &scolor);
+        gnome_bg_set_rgba (item->priv->bg, item->priv->shading, &pcolor, &scolor);
         gnome_bg_set_placement (item->priv->bg, item->priv->placement);
 }
 
